packages/vscode-extension/src/webview/views/PreviewView.tsx

```diff
@@ -147,17 +148,32 @@ function PreviewView() {
 
   function onMouseDown(e: MouseEvent<HTMLDivElement>) {
     e.preventDefault();
+    const elementName = (e.target as Element).className;
+    if (elementName !== "phone-wrapper") {
```

Comment:
This adds another layer of indirection as we rely on a specific class name to be used by a specific element that the component we render here renders – this makes it very complicated to follow in code and I wonder if there better way to avoid this.

On top of this, I wonder if this doesn't break the swipe from edge functionality that we have, where you can start swipe gesture at the phone frame and drag front the side (i.e. to go back on iOS).

One way I was hoping this could be solved was by using [contains()](https://developer.mozilla.org/en-US/docs/Web/API/Node/contains) along with react refs

Reply:
The simplest solution is to remove `onMouseDown` from PreviewView. It was causing unwanted touch down signals. Removing it should fix the issue if we want to start single and multitouch gestures from the edge of the phone rather than outside the touch area.